### DISCUSSION TITLE
Add Social Media account

### DISCUSSION THREAD

**Discussion by christianh814** [open-gitops/project#132] [Polls]
Simple question. Some background:

We currently have a Twitter and LinkedIn account where we share information publicly. We heavily use this account for GitOpsCon as well.

With the uncertainty of Twitter, should we create a Mastodon account somewhere? And if so where? There are a pile of servers (some not accepting new signups)

https://joinmastodon.org/servers

**Comment by christianh814** [comment]
I think https://fosstodon.org/ is the natural one. Not sure if there are better ones. I hope the CNCF creates one (probably won't happen)

**Reply by roberthstrand** [reply]
That sounds like a good plan, no other server that I've seen has been a better match.

**Comment by roberthstrand** [comment]
One alternative is [cloud-native.social](https://cloud-native.social), which is hosted by Civo.
